drafts/20003.adoc

```diff
@@ -0,0 +1,459 @@
+---
+layout: post
+title: "New Kafka Client API allowing Kafka specific message properties in Open Liberty 20.0.0.3"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/tomjenningss
+author_github: https://github.com/tomjenningss
+seo-title: New Kafka Client API allowing Kafka specific message properties in Open Liberty 20.0.0.3
+seo-description: Yu can use the Kafka client API. Allowing users to access Kafka specific message properties such as `message key` and `message headers`, not just the payload of the specific message. Also, you can now add the SameSite Cookie on the session, with three values on the SameSite attribute.
+---
+
+= New Kafka Client API allowing Kafka specific message properties in Open Liberty 20.0.0.3
```

Comment:
Maybe "New Kafka Client API to access specific message properties in Open Liberty 20.0.0.3" ? Bit more concrete than "allowing".